services/src/main/java/org/exoplatform/news/service/impl/NewsServiceImplV2.java

```diff
@@ -471,7 +489,7 @@ private News createDraftArticleForNewPage(News draftArticle, String pageOwnerId,
 
   private Page getArticlesRootPage(String ownerId) {
     List<Page> notes =
-                     noteService.getNotesOfWiki("group", ownerId)
+                     noteService.getNotesOfWiki(WikiType.GROUP.name().toLowerCase(), ownerId)
```

Comment:
here I think that we can retrieve the news root page directly with its name without looping